### DISCUSSION TITLE
Preventing line breaks in inline equations

### DISCUSSION THREAD

**Discussion by jeanas** [typst/typst#4751] [Q&A]
By default, Typst can insert line breaks inside inline equations, which I find ugly:

![image](https://github.com/user-attachments/assets/a26397cc-b1bc-4a47-9bb4-1d7f9463404a)

I tried

```
#show math.equation: box
```

This works but has an undesirable side effect: now the baseline of the text is no longer aligned to the baseline of the equation but to its vertical center instead, which is ugly too:

**Edit**: uploaded the wrong image here.

![image](https://github.com/user-attachments/assets/eccd1984-14f1-47b6-b03f-179de098b969)

How can I prevent line breaks without this side effect?

**Comment by Enivex** [comment]
This would be a bug. ~~Not sure which commit fixed it, but the baseline is correct on main.~~

Edit: Not fixed in general, just (coincidentally) almost correct in this case.

**Comment by Enter-tainer** [comment]
it would be useful if we can get baseline in typst-script
